refactor/urls.js

```diff
@@ -1,16 +1,25 @@
 // Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  let url = "/tasks";
+  let params = new URLSearchParams();
+  if (dev) {
+    params.append("status", "ACTIVE");
+    params.append("dev", "true");
+    params.append("size", "20");
+  }
 
-    if (tasks.nextTasks) {
-        url += '?hasNext=true';
-    }
+  if (tasks.nextTasks) {
+    params.append("hasNext", "true");
+  }
 
-    if (tasks.prevTasks) {
-        url = '/tasks?hasPrev=true';
-    }
-    return { url };
+  if (tasks.prevTasks) {
+    params.append("hasPrev", "true");
+  }
+
+  const paramsString = params.toString();
+  if (paramsString) {
+    url += `?${paramsString}`;
```

Comment:
Possible issues in future. Please fix.

Reply:
The possible issue I can find here is "params could be undefined" which leads to a .toString() error
But since I assign params with new URLSearchParams, even after .toString(), it doesn't throw any error!

Is there any possible issue that I could have missed, please let me know your feedback

Reply:
Hi @ankushdharkar 

I have fixed it to find if we have any params using params.size > 0